app/src/main/java/ch/nevis/exampleapp/coroutines/dagger/ApplicationModule.kt

```diff
@@ -117,12 +140,21 @@ class ApplicationModule {
             .build()
     }
 
+    @Provides
+    fun provideAuthenticatorAllowlist(): List<String> = listOf(
```

Comment:
Is this intentionally hardcoded?
Is there maybe some kind of configuration file where this could be outsourced to?

Reply:
It's by design, in case of the Android example apps there are no such kind of configuration file, everything can be found in this class.